install/setup/setup-wizard.sh

```diff
@@ -50,14 +50,19 @@ if [ "$install_db" == "y"  ] || [ "$install_db" == "Y" ] ; then
 
     # Only supporting MySql at this time
     database_type=0
-    while [[ ! ${database_type} =~ ^[1-2]{1}$ ]]; do
+    while [[ ! ${database_type} =~ ^[1-3]{1}$ ]]; do
         echo "Which database (Enter the number)?"
         echo "1) MySQL"
         echo "2) PostgresSQL"
+        echo "3) SQLServer"
```

Comment:
I only tested it with Azure-hosted SQL Server. I'd assume that this will work with a standard SQL Server though. I'll correct the name.